src/main/java/org/mockito/ArgumentCaptor.java

```diff
@@ -100,12 +84,41 @@ private ArgumentCaptor(Class<? extends T> clazz) {
      * @return null or default values
      */
     public T capture() {
-        Mockito.argThat(capturingMatcher);
+        return captureIf(Any.ANY);
```

Comment:
no the implementation of capture()
Am 19.03.2016 22:07 schrieb "terebesirobert" <notifications@github.com>:

> In src/main/java/org/mockito/ArgumentCaptor.java
> <https://github.com/mockito/mockito/pull/375#discussion_r56755136>:
>
> > @@ -100,12 +84,41 @@ private ArgumentCaptor(Class<? extends T> clazz) {
> >       * @return null or default values
> >       */
> >      public T capture() {
> > -        Mockito.argThat(capturingMatcher);
> > +        return captureIf(Any.ANY);
>
> You mean we should rewrite the forClass implementation?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/mockito/mockito/pull/375/files/8b9cc932314cf9ee479cd03e907fc4dd0a34cb24#r56755136>
>